extensions/amp-story/1.0/amp-story.js

```diff
@@ -2234,7 +2234,7 @@ export class AmpStory extends AMP.BaseElement {
     const historyNavigationPath =
       getHistoryState(this.win, HistoryState.NAVIGATION_PATH);
     if (historyNavigationPath) {
-      this.storyNavigationPath_ = historyNavigationPath;
+      this.storyNavigationPath_.push(historyNavigationPath);
```

Comment:
@rsimha this is not logically equivalent, and I think could break history in stories.  A better bet might be `concat`, but before we go there, what was wrong with what was there before?

Reply:
@newmuis Without this change:
```js
extensions/amp-story/1.0/amp-story.js:2237: ERROR - assignment to property storyNavigationPath_ of AmpStory
found   : string
required: (Array<string>|null)
      this.storyNavigationPath_ = historyNavigationPath;
      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
I've switched from `push` to `concat` in https://github.com/ampproject/amphtml/pull/21618/commits/66aadedf6c417aa2d25f55b5305edb006b9a760b. Let me know if this works.

Reply:
@newmuis Is this version okay?

Reply:
@newmuis I revisited this code. It's clear that `this.storyNavigationPath_` must be an `{Array<string>}`. This function is `initializeStoryNavigationPath_`, so I'd imagine `this.storyNavigationPath_` is `[]` at this point, so we'd need to add `historyNavigationPath` to an empty array.

Reply:
As discussed in person, I've undone the code changes and fixed the type info. Done in 98a88e7.